src/urh/dev/native/lib/cbladerf.pxd

```diff
@@ -94,16 +99,17 @@ cdef extern from "libbladeRF.h":
         int bladerf_set_bandwidth(bladerf *dev, bladerf_module module, unsigned int bandwidth, unsigned int *actual)
         int bladerf_get_bandwidth(bladerf *dev, bladerf_module module, unsigned int *bandwidth)
 
-    IF BLADERF_API_VERSION >= 2:
+    IF BLADERF_API_VERSION >= 1.91:
```

Comment:
I am not very happy with this change. BladeRF guys might have a reason to announce the feature only in the 2.0 version of their API. Can we put the requirement back to 2.0?

Moreover, can we put all conditional definitions in this file under this ``` IF BLADERF_API_VERSION >= 2 ``` it will be easier to review.

Reply:
@jopohl, the changes regarding setting up bias tee has been added when `BLADERF_API_VERSION >=2`. 

I had to modify some code regarding `bladerf_set_frequency / bladerf_get_frequency` methods since I made the  compilation tests for both `BLADERF_API_VERSION >= 1.91` and `BLADERF_API_VERSION >= 2`, where the first option was throwing me some errors in order to compile. 

I've update the code trying to follow feedback as close as possible I can. But certainly I had to make some changes when `BLADERF_API_VERSION >= 1.91`. 

If you want I can squash my commits from the first one so that it would be easier for you to see changes from latest upstream commit. 

Let me know,

Reply:
No worries, GitHub does a pretty good job at showing diffs. I will squash the commits once this is done when merging to master.